ui/app/AppLayouts/CommunitiesPortal/popups/DiscordImportProgressContents.qml

```diff
@@ -0,0 +1,313 @@
+                    model: SortFilterProxyModel {
+                        sourceModel: root.store.discordImportErrors
+                        filters: ValueFilter { roleName: "taskId"; value: subtaskDelegate.type }
+                        sorters: RoleSorter { roleName: "code"; sortOrder: Qt.DescendingOrder } // errors first
+                    }
+                    delegate: IssuePill {
+                        Layout.fillWidth: true
+                        horizontalPadding: 12
+                        verticalPadding: 8
+                        bgCornerRadius: 8
+                        visible: text
+                        type: model.code === Constants.DiscordImportErrorCode.Error ? IssuePill.Type.Error : IssuePill.Type.Warning
+                        text: model.message
+                    }
+                }
+            }
+            StatusDialogDivider {
+                Layout.fillWidth: true
+                Layout.leftMargin: -24 // compensate for Control.horizontalPadding -> full width
+                Layout.rightMargin: -24 // compensate for Control.horizontalPadding -> full width
+                visible: !parent.Positioner.isLastItem
+            }
+        }
+    }
+
+    ColumnLayout {
+        width: parent.width
+        spacing: 20
+
+        RowLayout {
+            Layout.fillWidth: true
+            spacing: 12
+
+            Image {
+                Layout.preferredWidth: 36
+                Layout.preferredHeight: 36
+                sourceSize: Qt.size(36, 36)
+                source: Style.svg("contact") // TODO community icon
+            }
+            StatusBaseText {
+                Layout.fillWidth: true
+                font.pixelSize: 15
+                text: {
+                    switch (d.status) {
+                    case DiscordImportProgressContents.ImportStatus.InProgress:
+                        return qsTr("Importing ‘%1’ from Discord...").arg(root.store.discordImportCommunityName)
+                    case DiscordImportProgressContents.ImportStatus.Stopped:
+                        return qsTr("Importing ‘%1’ from Discord stopped...").arg(root.store.discordImportCommunityName)
+                    case DiscordImportProgressContents.ImportStatus.StoppedWithErrors:
+                        return qsTr("Importing ‘%1’ stopped due to a critical issue...").arg(root.store.discordImportCommunityName)
+                    case DiscordImportProgressContents.ImportStatus.CompletedWithWarnings:
+                        return qsTr("‘%1’ was imported with %n issue(s).", "", root.store.discordImportWarningsCount).arg(root.store.discordImportCommunityName)
+                    case DiscordImportProgressContents.ImportStatus.CompletedSuccessfully:
+                        return qsTr("‘%1’ was successfully imported from Discord.").arg(root.store.discordImportCommunityName)
+                    default:
+                        return qsTr("Your Discord community import is in-progress...")
```

Comment:
I might be wrong, but I don't think `in progress` has a `- `

Reply:
Fixed!